uaclient/cli.py

```diff
@@ -880,9 +880,9 @@ def action_status(args, cfg):
     show_beta = args.all if args else False
     status = cfg.status(show_beta=show_beta)
     active_value = ua_status.UserFacingConfigStatus.ACTIVE.value
-    config_active = bool(status["configStatus"] == active_value)
+    config_active = bool(status["execution_status"] == active_value)
```

Comment:
This is another part of the schema that we will need to handle in the `postinst` schema patch script